packages/sources/ion.au/src/transport/total_reserve.ts

```diff
@@ -0,0 +1,55 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { BaseEndpointTypes } from '../endpoint/total_reserve'
+
+export interface ResponseSchema {
+  total_reserve: number
+  errorMessage?: string
+}
+
+export type HttpTransportTypes = BaseEndpointTypes & {
+  Provider: {
+    RequestBody: never
+    ResponseBody: ResponseSchema
+  }
+}
+export const httpTransport = new HttpTransport<HttpTransportTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((param) => {
+      return {
+        params: [param],
+        request: {
+          baseURL: config.API_ENDPOINT,
+          url: '/',
+          headers: {},
+          params: {},
+        },
+      }
+    })
+  },
+  parseResponse: (params, response) => {
+    if (!response.data) {
```

Comment:
Could you also add a separate validation on `response.data.errorMessage` before this one in case an error message exists. We can also return a 502 error code for that